shared/components/map.js

```diff
@@ -276,8 +287,12 @@ export default class Map extends Component {
     return null;
   }
 
-  renderUserIndicatorMarker() {
-    if (this.props.zoomLevel <= 18 && this.props.zoomLevel > 17) {
+  checkUserZoomLevel() {
+    return (this.props.zoomLevel <= 18 && this.props.zoomLevel > 17);

```

Comment:
no zoomLevel we are having can have floating points. it is actually very hard to get exact zoom level of 18